### DISCUSSION TITLE
Montage and Overlay

### DISCUSSION THREAD

**Discussion by PedroXSQueiroz** [Sixze/ALS-Refactored#445] [Q&A]
My intenttion is add some dodge/evade animations. I'm playing them as montages and is working fine, but the overlays are not overriding the arms causing this strange effect:
   
![Montage+Overlay](https://github.com/Sixze/ALS-Refactored/assets/7254594/543b1e7a-424f-48af-9b28-589904a68a50)

I added that slot to play the animations and I thought the overlays would be able to overide the previous poses:
![image](https://github.com/Sixze/ALS-Refactored/assets/7254594/3b05bf09-7383-4c03-90f8-9b9ed1aa6efd)

Does anyone have any idea of what I'm doing wrong or some workaround I can do?

**Comment by Magic-Summer** [comment]
Hi @PedroXSQueiroz
It may be that your curves need to be tuned. As you see, your montage flows into `PostLocomotion` and then into `Layering` (which is controlled by curves) 
You may find some help in these posts from @Sixze :
[how layering basically works](https://github.com/Sixze/ALS-Refactored/discussions/19)
[Сurves control the blending of the overlay pose](url)
Good luck!
